### PR TITLE
adding setAuthorizationHeader method

### PR DESCRIPTION
**Pull request by stour**
Needed in order to [PUT document to VSTS extension storage REST API](https://github.com/stour/plugins/blob/CODENVY-106/plugin-github/codenvy-plugin-github-server/src/main/java/com/codenvy/plugin/github/server/VSTSWebhookService.java#L245).

### CODE REVIEW

**Comment by skabashnyuk** [comment]
@garagatyi @evoevodin ?

File: core/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/HttpJsonRequest.java [hunk 1]
```diff
@@ -122,6 +122,17 @@
     HttpJsonRequest addQueryParam(@NotNull String name, @NotNull Object value);
 
     /**
+     * Adds authorization header to the request.
+     *
+     * @param value
+     *         authorization header value
+     * @return this request instance
+     * @throws NullPointerException
+     *         when either name or value is null
```
Comment (voievodin) [inline core/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/HttpJsonRequest.java]: What `name` ?

**Comment by voievodin** [comment]
Looks good  :muscle:. But fix javadoc please

File: core/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/HttpJsonRequest.java (hunk 1 above)
Comment (garagatyi) [inline core/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/HttpJsonRequest.java]: +1

**Comment by garagatyi** [comment]
LGTM

**Comment by skabashnyuk** [comment]
ok. javadoc please

**Comment by stour** [comment]
javadoc fixed.

**Comment by skabashnyuk** [comment]
ok. Will you merge? Or you need someone to merge?

**Comment by benoitf** [comment]
better if someone merge because he's off

**Pull request merged**